integration_tests/test_runner/src/bootstrapping.rs

```diff
@@ -0,0 +1,186 @@
+use core::str::FromStr;
+use std::thread;
+
+use crate::utils::ValidatorKeys;
+use crate::utils::OPERATION_TIMEOUT;
+use crate::utils::{
+    get_deposit, get_gravity_chain_id, get_ibc_chain_id, HERMES_CONFIG, RELAYER_ADDRESS,
+    RELAYER_MNEMONIC,
+};
+use deep_space::private_key::{CosmosPrivateKey, PrivateKey, DEFAULT_COSMOS_HD_PATH};
+use deep_space::Contact;
+use ibc::core::ics24_host::identifier::ChainId;
+use ibc_relayer::config::AddressType;
+use ibc_relayer::keyring::{HDPath, KeyRing, Store};
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::os::unix::io::{FromRawFd, IntoRawFd};
+use std::process::{Command, Stdio};
+
+/// Parses the output of the cosmoscli keys add command to import the private key
+fn parse_phrases(filename: &str) -> (Vec<CosmosPrivateKey>, Vec<String>) {
```

Comment:
eventually we may want to move some of this more boilerplate stuff to some upstream crate, but I don't see the need now and it would be actively harmful to flexibility. So maybe it's not the best idea?

Reply:
This will be done in another PR

Reply:
Will do this as part of a test env refactor soon